README.md

```diff
@@ -53,6 +65,24 @@ after(() => {
 
 ```
 
+### cy.setupPactHeaderAllowlist([header])
+Add a list of headers that will be included in a pact at test case level
+
+**Example**
+```js
+before(() => {
+    cy.setupPact('ui-consumer', 'api-provider')
+    cy.intercept('GET', '/users').as('getAllUsers')
+    cy.setupPactHeaderAllowlist([''])
```

Comment:
In this example, does it clear out the default allow list? It looks like you're adding a blank header as an allowed option, which seems confusing.